HowTos/Controller_Login_SAML_Config.rst

```diff
@@ -37,28 +39,87 @@ If you haven’t already deployed the Aviatrix controller, follow `the Controlle
 2.2 IDP Account
 ###############
 
-An IDP refers to an identity provider for SAML. This could be any provider that supports a SAML end point like `Okta <./UserSSL_VPN_Okta_SAML_Config.html>`__, `OneLogin <./UserSSL_VPN_OneLogin_SAML_Config.html>`__, `Google <./UserSSL_VPN_Google_SAML_Config.html>`__, `AWS SSO <./UserSSL_VPN_AWS_SSO_SAML_Config.html>`__, `Azure AD <./UserSSL_VPN_Azure_AD_SAML_Config.html>`__, Ping Identity, VmWare VIDM, ForgeRock's OpenAM etc. (The listed ones were tested). You will require administrator access to create IDP endpoints for SAML.
+An IDP refers to an identity provider for SAML. This could be any provider that supports a SAML end point like `Okta <./SAML_Integration_Okta_IDP.html>`__,
+`OneLogin <./SAML_Integration_OneLogin_IDP.html>`__, `Google <./SAML_Integration_Google_IDP.html>`__,
+`AWS SSO <./SAML_Integration_AWS_SSO_IDP.html>`__, and `Azure AD <./SAML_Integration_Azure_AD_IDP.html>`__.
+You will require administrator access to create IDP endpoints for SAML. Check `IDP-specific SAML Integration <#idp-integration>`__ to see a list of guides for supported IDP's
+
 
 
 3. Configuration Steps
 ----------------------
 
-Follow these steps to configure Aviatrix to authenticate against your Okta IDP:
+Follow these steps to configure Aviatrix to authenticate against IDP:
 
-1. Create `SAML App <#config-31>`__ for Aviatrix
-2. Retrieve `IDP Metadata <#config-32>`__
-#. Create Aviatrix `SAML SP Endpoint <#config-33>`__
-#. `Test the Integration <#config-34>`__ is Set Up Correctly
-#. `Validate <#config-35>`__
+  1. Create `temporary Aviatrix SP Endpoint <#config-31>`__ for Aviatrix controller
+  2. Create `SAML IDP App <#config-32>`__ with specific IDP
+  #. Retrieve `IDP Metadata <#config-33>`__ from IDP
+  #. Update `Aviatrix SP Endpoint <#config-34>`__ with IDP metadata
+  #. `Test the Integration <#config-35>`__ is set up correctly
+  #. `Validate <#config-36>`__
 
 .. _Config_31:
 
-3.1 Create a SAML App for Aviatrix at the IDP
-#############################################
+3.1 Create temporary Aviatrix SP Endpoint
+#########################################
+.. note::
+
+   This step is usually completed by the Aviatrix admin.
+   This endpoint will be updated later on in the guide, at this step, we will be using placeholder values.
+   Choose an endpoint name for ``aviatrix_sp_name`` which will be used throughout the guide.
```

Comment:
what is aviatrix_sp_name. We dont need too many variables. Just call it endpoint name